womtool/src/main/scala/cromwell/pipeline/womtool/WomTool.scala

```diff
@@ -0,0 +1,78 @@
+package cromwell.pipeline.womtool
+
+import java.nio.file.{ Files, Paths }
+
+import cats.data.NonEmptyList
+import com.typesafe.config.ConfigFactory
+import common.Checked
+import cromwell.core.path.{ DefaultPath, DefaultPathBuilder, Path }
+import cromwell.languages.LanguageFactory
+import cromwell.languages.util.ImportResolver.ImportResolver
+import languages.cwl.CwlV1_0LanguageFactory
+import languages.wdl.biscayne.WdlBiscayneLanguageFactory
+import languages.wdl.draft2.WdlDraft2LanguageFactory
+import languages.wdl.draft3.WdlDraft3LanguageFactory
+import wom.executable.WomBundle
+import womtool.WomtoolMain.{ BadUsageTermination, SuccessfulTermination, Termination, UnsuccessfulTermination }
+import womtool.inputs.Inputs
+
+import scala.util.Try
+
+class WomTool extends WomToolAPI {
+
+  def validate(
+    content: String,
+    importResolvers: List[ImportResolver]
```

Comment:
It would be convenient to have `importResolvers` as an object field instead of a function parameter, I suggest we extract it:
```scala
class WomTool(val importResolvers: List[ImportResolver]) extends WomToolAPI {
  /* ... */
}
```

Reply:
finished